### PR TITLE
COMP: Fix windows link error ensuring TCGAUtilities symbols are exported.

### LINKED ISSUE #28: ShortCutCoreSegmenter::RefineShortCut: Fix windows link error
```
Error   6   error LNK2019: unresolved external symbol "void __cdecl 
TCGA::FindBoundingBoxFromMask(class cv::Mat const &,class cv::Rect_<int> &)" 
(?FindBoundingBoxFromMask@TCGA@@YAXAEBVMat@cv@@AEAV?$Rect_@H@3@@Z) 
referenced in function "public: void __cdecl ShortCutCoreSegmenter::RefineShortCut(void)" 
(?RefineShortCut@ShortCutCoreSegmenter@@QEAAXXZ)    C:\D\N\S-1-E-b\SlicerPathology-
build\ShortCutCore\Logic\ShortCutCoreSegmenter.obj  vtkSlicerShortCutCoreModuleLogic

```

### PR DESCRIPTION
**Pull request by jcfr**
Fixes #28

### CODE REVIEW

**Comment by jcfr** [comment]
xref Slicer/Slicer#485

**Comment by jcfr** [comment]
Cc: @fedorov @naucoin

File: QuickTCGA/Logic/TCGAUtilities.h [hunk 1]
```diff
@@ -23,6 +23,8 @@
 
 #include <csignal>
 
+#include "vtkSlicerQuickTCGAModuleLogicExport.h"
```
Comment (fedorov) [inline QuickTCGA/Logic/TCGAUtilities.h:26]: Is this file auto-generated? I don't see it in the repo

**Comment by fedorov** [comment]
@ebremer considering @jcfr is the key contributor to the SlicerPathology work, could you please add him to the SBU-BMI organization?

File: QuickTCGA/Logic/TCGAUtilities.h (hunk 1 above)
Comment (jcfr) [inline QuickTCGA/Logic/TCGAUtilities.h:26]: This is a standard file configured in the build directory of all Slicer libraries.

Comment (jcfr) [inline QuickTCGA/Logic/TCGAUtilities.h:26]: It is configured with the value specified here: https://github.com/SBU-BMI/SlicerPathology/blob/f07fbc8d7d7065b942bb3cae94a6aa4d36a1c3a9/QuickTCGA/Logic/CMakeLists.txt#L46

Comment (fedorov) [inline QuickTCGA/Logic/TCGAUtilities.h:26]: Thanks for the clarification, I thought it is a standard file, but wanted to confirm.

**Pull request merged**